experimental/packages/otlp-exporter-base/src/configuration/otlp-http-configuration.ts

```diff
@@ -61,14 +61,40 @@ function validateUserProvidedUrl(url: string | undefined): string | undefined {
   if (url == null) {
     return undefined;
   }
-  try {
-    new URL(url);
+
+  // If it looks like an absolute URL (contains ://), validate it strictly
+  if (url.includes('://')) {
+    try {
+      new URL(url);
+      return url;
+    } catch {
+      throw new Error(
+        `Configuration: Could not parse user-provided export URL: '${url}'`
+      );
+    }
+  }
+
+  // For relative URLs, do basic validation
+  // Allow paths that start with ./, ../, /, or are simple paths without spaces
+  if (url.match(/^(\.\.?\/|\/|[a-zA-Z0-9._-]+)/)) {
+    // Check for obvious invalid characters that would make it not a valid path
+    if (
```

Comment:
at this point, are ASCII replaced with their value? meaning if someone used `%20` would we see ` ` or `%20`?  Just wanna make sure it's properly encoded before this check

Reply:
They are not; I included a method to decode the URL before checking for problematic chars.